lib/compiler/src/engine/inner.rs

```diff
@@ -200,12 +200,38 @@ impl Engine {
         Ok(Arc::new(Artifact::deserialize(self, bytes)?))
     }
 
+    /// Deserializes a WebAssembly module
     #[cfg(not(target_arch = "wasm32"))]
+    pub fn deserialize_checked(&self, bytes: &[u8]) -> Result<Arc<Artifact>, DeserializeError> {
+        #[cfg(not(target_arch = "wasm32"))]
```

Comment:
the `cfg` is probably not usefull here.

Reply:
As above, just an artifact of the revert.